packages/dapp/src/const/MenuConfig/ConnectedMenu/Connected-SubMenu-Chains.tsx

```diff
@@ -6,16 +6,25 @@ import { useWallet } from '../../../providers/WalletProvider';
 import { MenuListItem } from '../../../types';
 
 const ConnectedSubMenuChains = () => {
-  const { account, usedWallet, disconnect, switchChain } = useWallet();
-  const { chains, isSuccess } = useChainInfos();
+  const { account, switchChain } = useWallet();
+  const { chains } = useChainInfos();
   const activeChain = useMemo(
     () => chains.find((chainEl: Chain) => chainEl.id === account.chainId),
-    [chains.length, account.chainId],
+    [chains, account.chainId],
   );
 
+  let availableChains = chains;
+
+  if ((import.meta as any).env.VITE_LIFI_SHOW_TESTNET) {
```

Comment:
You could simplify that check by checking the build `mode`. You added `vite --mode testnet` to the package.json scripts